Makefile.include

```diff
@@ -230,11 +230,14 @@ GLOBAL_GOALS += buildtest \
                 info-boards-features-missing \
                 info-boards-supported \
                 info-buildsizes info-buildsizes-diff \
+                info-kconfig-boards-supported \
+                info-kconfig-board-supported-% \
                 #
 
 ifneq (, $(filter $(GLOBAL_GOALS), $(MAKECMDGOALS)))
-  include $(RIOTMAKE)/info-global.inc.mk
-  include $(RIOTMAKE)/buildtests.inc.mk
+    include $(RIOTMAKE)/info-kconfig.inc.mk
+    include $(RIOTMAKE)/buildtests.inc.mk
+    include $(RIOTMAKE)/info-global.inc.mk
```

Comment:
I still need to modify this so it does not run the current check when checking with Kconfig.